src/test/kotlin/com/igorwojda/list/search/linearsearch/challenge.kt

```diff
@@ -4,7 +4,7 @@ import org.amshove.kluent.shouldBeEqualTo
 import org.junit.jupiter.api.Test
 
 private fun getIndex(list: List<String>, str: String): Int {
-    TODO("not implemented")
+    return list.indexOf(str)
```

Comment:
The requirement specifies: Don't use any Kotlin build-in methods that are directly returning index of element like `indexOf` / `lastIndexOf` /
`indexOfFirst` / `indexOfLast`, etc.